packages/office-ui-fabric-react/src/components/TeachingDialog/examples/TeachingDialog.Basic.Example.tsx

```diff
@@ -0,0 +1,74 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+
+import {
+  TeachingDialog
+} from '../index';
+
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { IImageProps } from 'office-ui-fabric-react/lib/Image';
+import { ITeachingDialogViewProps } from '../TeachingDialogView.Props';
+
+export interface ITeachingDialogBasicExampleState {
+  isTeachingDialogVisible?: boolean;
+}
+
+export class TeachingDialogBasicExample extends React.Component<any, ITeachingDialogBasicExampleState> {
+
+  private _menuButtonElement: HTMLElement;
+  private x: boolean;
+  public constructor() {
+    super();
+    this._onDismiss = this._onDismiss.bind(this);
+    this.x = false;
+    this.state = {
+      isTeachingDialogVisible: false,
+    };
+  }
+
+  public render() {
+    const baseProductionCdnUrl = 'https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets';
+    let imageProps: IImageProps = { src: 'odsp-media/images/spfirstrun/Activity.gif' } as IImageProps;
+    let viewProps: ITeachingDialogViewProps[] =
+      [{
+        headline: 'Welcome',
+        leftButtonText: 'Discard',
+        rightButtonText: 'Next',
+        onLeftButton: this._onDismiss,
+        isLeftButtonLight: true,
+        title: 'Lorem ipsum',
+        textContent: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Facere, nulla, ipsum? Molestiae quis aliquam magni harum non?',
+        image: [baseProductionCdnUrl, 'teachingDialog.gif'].join('/'),
+      } as ITeachingDialogViewProps,
+      {
+        leftButtonText: 'Previous',
+        rightButtonText: 'Done',
+        onRightButton: this._onDismiss,
+        title: 'Lorem ipsum 2',
+        textContent: 'Lorem ipsum 2 dolor sit amet, consectetur adipisicing elit. Facere, nulla, ipsum? Molestiae quis aliquam magni harum non?',
+        image: [baseProductionCdnUrl, 'teachingDialog.gif'].join('/'),
+      } as ITeachingDialogViewProps];
+
+    return (
+      <div className='ms-TeachingDialogExample'>
+        <span className='ms-TeachingDialogBasicExample-buttonArea' ref={ (menuButton) => this._menuButtonElement = menuButton }>
```

Comment:
HTMLSpanElement | Null is not assignable to type HTMLElemement. Please change the type of this._menuButtonElement to HTMLSpanElement.